tools/remark-preset-lint-node/index.js

```diff
@@ -42,8 +42,10 @@ module.exports.plugins = [
   [
     require('remark-lint-prohibited-strings'),
     [
-      { no: 'v8', yes: 'V8' },
-      { no: 'Javascript', yes: 'JavaScript' }
+      { no: 'Github', yes: 'GitHub' },
```

Comment:
Isn't `tools/remark-preset-lint-node/index.js` technically provided by watilde/remark-preset-lint-node, and shouldn't be edited downstream? Ref https://github.com/nodejs/node/pull/17441

Reply:
It's a small thing, but I'd prefer that the version downstream get synchronized with this version (after this lands) rather than landing downstream first. If it lands downstream first, there's a race condition where new errors can creep in here before the updated version lands here. Not too big a deal for us as we can fix the problematic text, but potentially irksome for anyone installing the preset locally and suddenly getting errors on the Node.js master branch.

The problem with the approach I suggest above is that the `package.json` becomes sorta kinda meaningless. Like, we're not running the version specified in the `package.json`. We're running a modified version and we need to update it again when this gets published upstream.

Anyway, I don't feel *too* strongly about this and can be persuaded that this should wait for downstream rather than the other way around if people feel strongly about it. Related: We should probably move the repo to the nodejs org and allow a few more folks to publish it, assuming @watilde has no problem with that.

Reply:
>It's a small thing, but I'd prefer that the version downstream get synchronized with this version (after this lands) rather than landing downstream first.

Ok, fine by me.

My other point (and why I put a link to my PR) was that this (very small) module should live within the nodejs/node repo, so that we don't have to worry about upstreaming/downstreaming changes and publishing versions to npm.